packages/circuit-ui/components/SelectorGroup/SelectorGroup.stories.tsx

```diff
@@ -29,6 +29,7 @@ const baseArgs = {
     { children: 'Apple', value: 'apple', noMargin: true },
     { children: 'Banana', value: 'banana', noMargin: true },
     { children: 'Mango', value: 'mango', noMargin: true },
+    { children: 'I like all fruits', value: 'all', noMargin: true },
```

Comment:
Added an option with multiple lines on the Base story as mentioned, to cover with Chromatic. Not worth maintaining an extra story 🙂